dtbase/core/db.py

```diff
@@ -126,7 +126,7 @@ def drop_db(conn_string, db_name):
                 "pid_column": pid_column,
                 "database": db_name,
             }
-            connection.execute(sqla.text(text))
+            connection.execute(text)
```

Comment:
Interestingly, the tests pass but then the CI fails when dropping the database. 

I noticed this change in drop_db which is a code change rather than a type annotation. Could this be causing the test error?